### PR TITLE
etcd status monitoring

### PR DESCRIPTION
**Pull request by mmahut**
etcd status check retrieves the store status from etcd API and collect interesting items which can be plotted into graphs.

We need to bind mount the etcd client certs to access the API.

### CODE REVIEW

**Comment by mmahut** [comment]
Tested locally and all appear to be working fine.

@twiest ready for review, please take a look.

**Comment by twiest** [comment]
@mmahut please rebase onto the latest master.

File: scripts/monitoring/cron-send-etcd-status.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+def main():
+    ''' Get data from etcd API
+    '''
+
+    API_HOST = 'https://localhost:4001/v2/stats/store'
+    SSL_CLIENT_CERT = '/etc/openshift/master/master.etcd-client.crt'
+    SSL_CLIENT_KEY = '/etc/openshift/master/master.etcd-client.key'
+
+    # Fetch the store statics from API
+    request = requests.get(API_HOST, cert=(SSL_CLIENT_CERT, SSL_CLIENT_KEY), verify=False)
```
Comment (twiest) [inline scripts/monitoring/cron-send-etcd-status.py]: I would suggest adding another zabbix item called "openshift.master.etcd.ping" and when you're able to ping etcd, then send a 1. When you're not able to ping etcd, send a 0.

Currently, what happens here when etcd is down?

I would also place this in a try / catch, just to make sure.

That's the kind of logic that I do in the docker info gathering script.

See here:
https://github.com/openshift/openshift-tools/blob/master/docker/oso-rhel7-zagg-client/monitoring_scripts/cron-send-docker-metrics.py#L156-L180

**Comment by twiest** [comment]
@mmahut good job on this, it's looking very good.

File: scripts/monitoring/cron-send-etcd-status.py (hunk 1 above)
Comment (mmahut) [inline scripts/monitoring/cron-send-etcd-status.py]: Right now the ops-runner will fail - but you are right, I will catch that exception and send it as an item, so we can alert high on it if it really fails.

Comment (twiest) [inline scripts/monitoring/cron-send-etcd-status.py]: Yeah, so all ops-runner failures are considered bugs in the script. ops-runner is just there to alert us to the bugs. We never want to rely on ops-runner for valid failure cases.

Comment (mmahut) [inline scripts/monitoring/cron-send-etcd-status.py]: Fixed and rebased.
```
[CTR][root@e02d0d932ad6 ~]$ /usr/bin/cron-send-etcd-status
ERROR talking to etcd API: ('Connection aborted.', error(111, 'Connection refused'))
```

**Comment by mmahut** [comment]
@twiest PTAL

**Comment by twiest** [comment]
@mmahut Looks great!  :+1:

**Pull request merged**